scripts/generate-available-permission-types-table.py

```diff
@@ -46,31 +47,37 @@
     ResourceType.TRACE,
 ]
 
+def add_lines_for_permission_set(resource_title, permission_types, lines):
+    lines.append('%s' % (resource_title))
```

Comment:
Why format a single var like this?
```suggestion
    lines.append(resource_title)
```
Or if we do need to format:
```suggestion
    lines.append(f"{resource_title}")
```

Reply:
Oh. I see, you just moved what was already there.